### PR TITLE
CORS support mentioned and URL updated

### PR DESCRIPTION
**Pull request by ipapi-co**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -531,7 +531,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Hong Kong GeoData Store](https://geodata.gov.hk/gs/) | API for accessing geo-data of Hong Kong | No | Yes | Unknown |
 | [IP 2 Country](https://ip2country.info) | Map an IP to a country | No | Yes | Unknown |
 | [IP Address Details](https://ipinfo.io/) | Find geolocation with ip address | No | Yes | Unknown |
-| [IP Location](https://ipapi.co/api/#introduction) | Find IP address location information | No | Yes | Unknown |
+| [IP Location](https://ipapi.co/) | Find IP address location information | No | Yes | Yes |
```
Comment (pawelbr) [inline README.md:534]:
```suggestion
| [IP Location](https://ipapi.co/api#introduction) | Find IP address location information | No | Yes | Yes |
```

**Review by pawelbr — CHANGES_REQUESTED** [review]
Thanks for the contribution @ipapi-co 
But the previous link is fine no need to change it.
Please commit the suggestions stated below and don't forget to squash it.

**Comment by ipapi-co** [comment]
Getting an error with squash. We will close this PR & re-submit with the suggested changes. Thank you for your help.

**Comment by pawelbr** [comment]
Sure @ipapi-co 

You're welcome ☺️

**Pull request closed without merging**